pkg/provisioner/ironic/ironic.go

```diff
@@ -931,6 +931,12 @@ func (p *ironicProvisioner) Deprovision(deleteIt bool) (result provisioner.Resul
 			nodes.ProvisionStateOpts{Target: nodes.TargetManage},
 		)
 
+	case nodes.Inpsecting, nodes.InspectWait:
+		p.log.Info("waiting for inspection to complete")
```

Comment:
FYI inspection can be aborted

Reply:
Based on https://docs.openstack.org/ironic/latest/_images/states.svg it looks like I have to tell it to "abort" and then the node will enter the "inspect failed" state and I can then "manage" it to move it back to manageable. Is that right?

Reply:
The latest version does try to abort. It seems to be told the node is busy a lot, but it does eventually work.

Reply:
@dhellmann that is correct